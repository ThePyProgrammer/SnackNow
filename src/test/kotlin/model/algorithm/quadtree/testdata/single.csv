-0.01,0.01,0.01,-0.01
1,2
0,0
-0.01,0.01,0.01,-0.01