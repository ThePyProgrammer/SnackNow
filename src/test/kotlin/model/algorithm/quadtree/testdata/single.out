1) Queried (-0.01000, 0.01000) to (0.01000, -0.01000) and received 1 result(s):
[(0.00000, 0.00000)]
